RHIDP-12608/12573: MCP doc updates for 1.10 (#2168)
redhat-developer/red-hat-developers-documentation-rhdh: modules/integrate_interacting-with-model-context-protocol-tools-for-rhdh/proc-configure-model-context-protocol-in-rhdh.adoc

----
====

. Register the MCP tools that you install as a plugin source in your `{my-app-config-file}` file:

> ** To use the installed `extras` plugins, you must register them as a plugin sources:

Reviewer: ```suggestion
** To use the installed `extras` plugins, you must register them as plugin sources:
```

+
[source,yaml,subs="+attributes,+quotes"]
----
backend:
  actions:
    pluginSources:
      - software-catalog-mcp-extras
      - techdocs-mcp-extras
      - scaffolder-mcp-extras
----

> ** To use the built-in {backstage} catalog tools for fetching entities and registering or unregistering components, you must register `catalog` or `scaffolder` in your plugin source:

Reviewer: ```suggestion
** To use the built-in {backstage} catalog tools for fetching entities and registering or unregistering components, you must register `catalog` and `scaffolder` in your plugin sources:
```
Using "or" suggests exclusive choice, however the example shows both.

Author: Both these plugins are option. Rewrote to make sense.

+
[source,yaml,subs="+attributes,+quotes"]
----
backend:
  actions:
    pluginSources:
      - catalog
      - scaffolder
----

> ** If you are using an OpenAI models with the {product-very-short} MCP server, you must also add the following section to your `{my-app-config-file}` file:

Reviewer: ```suggestion
** If you are using OpenAI models with the {product-very-short} MCP server, add the following configuration to your `{my-app-config-file}` file to disable namespaced toolnames:
```
